packages/scripts/CHANGELOG.md

```diff
@@ -1,11 +1,13 @@
 ## Master
 
+### Enhancements
+- Bundle analysis now runs with module concatenation disabled. This represents the size of individual modules more accurately, at the cost of not providing an exact byte-for-byte match to the final size in the production chunk.
```

Comment:
```suggestion
- Bundle analysis in `build` script now runs with module concatenation disabled. This represents the size of individual modules more accurately, at the cost of not providing an exact byte-for-byte match to the final size in the production chunk.
```

I think it only impacts `build` script because `start` uses development mode.